lib/experimental/register-webfonts-from-theme-json.php

```diff
@@ -64,6 +89,7 @@ function gutenberg_register_webfonts_from_theme_json() {
 	}
 	foreach ( $webfonts as $webfont ) {
 		wp_webfonts()->register_webfont( $webfont );
+		wp_webfonts()->enqueue_webfont( $webfont['font-family'] );
```

Comment:
Does this now enqueue all variation webfonts (including unused) on the frontend for folks visiting the site?

Reply:
Only when the variation is actually active - so the webfonts are used. It doesn't enqueue webfonts from unused variations.

Reply:
awesome! thanks for the quick response! 😁